### ISSUE TITLE
Extension Options

### ISSUE THREAD

**Issue by smaragdus** [nbeaver/save-tabs-as-html#1]
This extension is exactly what I need but I would appreciate if there are some options which would allow the user to define the HTML file which would be generated. For example I do not need the technical information which is currently saved with the tabs:

- Number of tabs
- User agent
- Date
- ISO date
- Locale date
- JSON date
- YYYY-MM-DD
- Milliseconds since Unix epoch

So there might be an option (check box) whether to save this information or not and thus the user would not need to manually edit the HTML file.

Also I would prefer single line output for tab, for example:

[nbeaver/save-tabs-as-html: Webextension to save the title and URLs of open tabs as an HTML file.](https://github.com/nbeaver/save-tabs-as-html)

in stead of:

[nbeaver/save-tabs-as-html: Webextension to save the title and URLs of open tabs as an HTML file.](https://github.com/nbeaver/save-tabs-as-html)
-----https://github.com/nbeaver/save-tabs-as-html

Additional option would be numbered tabs, an example using somewhat similar extension ([TabToLink](https://chrome.google.com/webstore/detail/tabtolink/fdbgbkipjopaoioobnhceghnocdoghpg)), which produces a neat output:

![TabToLink 1 8 0 - 2021-12-07 - 002](https://user-images.githubusercontent.com/9025757/144933985-5a8f880b-8c30-4b26-aecd-fd27761dc274.png)

So in short my suggestion is implementation of extension options where the user would be able to specify the format of the created HTML file. I suppose that all this may be implemented via check boxes in extension options.

**Comment by mzso** [comment]
I miss the ability to only save tabs of the current window. And apparently no other addon can save HTML.
